aria/cli/commands/services.py

```diff
@@ -38,6 +41,53 @@ def services():
     pass
 
 
+@services.command(name='show',
+                  short_help='Display service information')
+@aria.argument('service-name')
+@aria.options.verbose()
+@aria.options.show_all
+@aria.options.show_graph
+@aria.options.show_json
+@aria.options.show_yaml
+@aria.pass_model_storage
+@aria.pass_logger
+def show(service_name, model_storage, all, graph, json, yaml, logger):
+    """Show information for a specific service template
+
+    `SERVICE_NAME` is the name of the service to display information on.
+    """
+    logger.info('Showing service {0}...'.format(service_name))
+    service = model_storage.service.get_by_name(service_name)
+
+    if json or yaml:
+        all = True
```

Comment:
i'd rename the flag, it's both unclear and a python keyword. `show_all` or `all_info` might be better.